compiler-rt/lib/builtins/CMakeLists.txt

```diff
@@ -761,6 +761,12 @@ else ()
 
   append_list_if(COMPILER_RT_HAS_STD_C11_FLAG -std=c11 BUILTIN_CFLAGS)
 
+  # Don't link the the CRT on Windows
```

Comment:
I'd prefer to clarify the terminology here a bit; the builtins is a static library, so we're not linking anything anyway. Perhaps `# Don't embed directives for picking any specific CRT`?

Reply:
I made this change in my rebase before merging